### PR TITLE
Inline explore jupyter

### PR DESCRIPTION
**Pull request by abhishekpratapa**
Added Inline Jupyter Explore.

### CODE REVIEW

File: src/python/turicreate/data_structures/sframe.py [hunk 1]
```diff
@@ -2373,7 +2373,7 @@ def head(self, n=10):
         """
         return SFrame(_proxy=self.__proxy__.head(n))
 
-    def to_dataframe(self):
+    def to_dataframe(self, explore=False):
```
Comment (znation) [inline src/python/turicreate/data_structures/sframe.py]: Let's not make explore be a parameter on another method. It should stay as its own method.

File: src/python/turicreate/data_structures/sframe.py [hunk 2]
```diff
@@ -2385,11 +2385,18 @@ def to_dataframe(self):
...
         assert HAS_PANDAS, 'pandas is not installed.'
         df = pandas.DataFrame()
         for i in range(self.num_columns()):
             column_name = self.column_names()[i]
-            df[column_name] = list(self[column_name])
+            if(self.column_types()[i] == _Image and explore):
+                pandas.set_option('display.max_colwidth', -1)
+                df[column_name] = [_image_resize(x[column_name])._to_pil_image() for x in self.select_columns([column_name])]
+            else:
+                df[column_name] = list(self[column_name])
```
Comment (znation) [inline src/python/turicreate/data_structures/sframe.py]: Is there not a proper way to put a PIL image inside a Pandas dataframe? That could allow us to reconcile these two codepaths and just have the dataframe actually contain the image.

Comment (znation) [reply src/python/turicreate/data_structures/sframe.py]: Check out https://www.kaggle.com/stassl/displaying-inline-images-in-pandas-dataframe for an example of storing the image directly in the dataframe.

File: src/python/turicreate/data_structures/sframe.py (hunk 1 above)
Comment (znation) [reply src/python/turicreate/data_structures/sframe.py]: Actually, if we can get the contents of the image into the dataframe, we can just do that all the time (and not add a new parameter here).

**Comment by abhishekpratapa** [comment]
Good?

**Pull request merged**